### PR TITLE
:pencil: Updated README.md and CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by SyedaMahamFahim**
## Description
This pull request addresses the following changes and improvements:
1. Updated the readme.md 
2. Updated the contributing.md file to include all the necessary links and instructions for contributing to this repository.

## Related Issues
- Closes #[1] 

Please feel free to suggest any necessary changes in the comment section, and I'll make sure to update as needed to help advance the project. Thank you.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +16,21 @@ The main objectives are :
 
 ![workflow](https://github.com/zenika-open-source/cheatsheets/blob/main/workflow.png?raw=true)
 
+## Live demo
+You can explore a live demo of this project [here](https://zenika-open-source.github.io/cheatsheets/)
+
+## Quick Start
+To create your very first cheatsheets, follow this [Quick Start Guide](https://zenika-open-source.github.io/cheatsheets/quick-start/) 
+
+## Template Confiugration
```
Comment (liollury) [inline README.md:25]: Here a small mistake for "Configuration" word

**Comment by SyedaMahamFahim** [comment]
My bad, I just fixed it.

**Comment by liollury** [comment]
Thanks @SyedaMahamFahim for contributing :)

**Comment by liollury** [comment]
@all-contributors please add @SyedaMahamFahim for documentation

**Pull request merged**